Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -11,7 +11,14 @@ public class IonObjectSerializer : IonSerializer<object>
         private const BindingFlags fieldBindings = BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Public;
         private readonly IonSerializer ionSerializer;
         private readonly IonSerializationOptions options;
-        private readonly Type targetType;
+        internal Type targetType;
```

Comment:
Why is this no visible and changeable? It should not be.

Reply:
The change was because the IonSerializer was changing the type. But that is no longer happening so this is back to `private readonly`.